src/services/jwt.service.js

```
@@ -0,0 +1,62 @@
+require('dotenv').config();
```

Minor: the code uses `require('dotenv').config()` here which is fine, but ensure environment variables (`JWT_KEY`, `JWT_REFRESH_KEY`, `JWT_RESET_KEY` and associated TTLs) are set in your environment. Missing keys will cause `jwt.sign`/`jwt.verify` to throw. If you expect to run tests or CI with missing envs, consider failing early with a clear error.